openshift-container-platform-4/policies/SI-System_and_Information_Integrity/component.yaml

```diff
@@ -134,13 +134,15 @@
 - control_key: SI-2 (6)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response for SI-2(6) is planned. Engineering progress can be
-        tracked via:
-
-        https://issues.redhat.com/browse/CMP-528
+        The customer will be responsible for removing previous versions of software
+        and firmware. This control is specific to organizational procedure, requiring
+        removal of software after an updated version has been installed. 
+        OpenShift can assist the customer with this process, using strategies
+        such as Blue/Green deployments to assist with replacing and removing old versions
+        of containerized software applications.
```

Comment:
Actually, SI-2(6) is applicable because ocp4 has the update service. OCP doesn't care about firmware, so that part for sure doesn't matter.

Reply:
@redhatrises Can we set up a call to discuss scope of this (and similar) controls? I want to understand the ambiguity of scope in the NIST guidance.

Added `oc adm update` command to the text of the item.